samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -627,31 +627,16 @@ The MCUboot-based targets that enable the ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KM
    The board targets based on the nRF54L SoC Series are currently the only targets that support the KMU-based key storage.
    See the :ref:`fast_pair_locator_tag_dfu` section of this sample documentation for the details regarding the supported signature algorithms, public key storage location and the signature key file.
 
-Using KMU requires the provisioning operation of the public key to be performed manually.
-Before performing the provisioning operation, you need to ensure that your board target is fully erased:
+Using KMU requires the provisioning operation of the public key.
+The KMU provisioning is performed automatically when the ``west flash`` command is executed with the ``--erase`` or ``--recover`` flag.
```

Comment:
Consider mentioning that it's because the sample enables ``SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE`` (as it's non-default and it used to work differently)

Reply:
Section rewritten to provide extra context